front-end/e-learning/src/components/categories/Create.js

```diff
@@ -31,13 +31,18 @@ const Create = ({ createCategory, categories, token, user }) => {
   const validate = () => {
     var no_error = true;
     if (title === "") {
-      setErrorTitle("Field is required!");
+      setErrorTitle("Title is required!");
+      no_error = false;
+    } else if (title.length > 255) {
+      setErrorTitle("Title is limited to 255 characters!");
```

Comment:
**Not a comment, just note. It's up to you if you want to update now, but we can update this in the future.**
Next time, we can create a constants file, where we can store all constants, like these, error messages, default string length etc. Pros are, it's easier to update and easier to locate if we want to update.

example
```
// constants.js
errorMessages: {
  titleRequired: 'Title is required',
  titleMaxLength: 'Title is limited to 255 characters!',
  ...
}

export { errorMessages, ...<other constants if there are> }
```

then to use, you can just simply import the errorMessges constants file and make use of them to set error messages to the screen.

Reply:
@AubreyV yes ma'am, noted on this. Thank you for the insights, I'll bare this in mind.